src/BlazorWebView/src/Maui/BlazorWebViewHandler.cs

```diff
@@ -46,7 +46,7 @@ public BlazorWebViewHandler(PropertyMapper? mapper) : base(mapper ?? BlazorWebVi
 		/// <param name="webView">The <see cref="IBlazorWebView"/>.</param>
 		public static void MapHostPage(BlazorWebViewHandler handler, IBlazorWebView webView)
 		{
-#if !NETSTANDARD
+#if !(NETSTANDARD || !PLATFORM)
```

Comment:
This is a weird case and probably can be simplified to just `#if !PLATFORM` but that is for later as this PR is a no-real-change PR.